content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -11,7 +11,7 @@ the sibling file _docker.adoc (used in index.adoc).
 ==== On macOS and Linux
 
 . Open up a terminal window.
-. Run the `jenkinsci/blueocean` image as a container in Docker using the
+. Run the `jenkinsci/jenkins` image as a container in Docker using the
```

Comment:
```suggestion
. Run the `jenkins/jenkins` image as a container in Docker using the
```

The jenkinsci/jenkins image has been [deprecated](jenkins/jenkins/).  We need to point them to the official [jenkins/jenkins/ image](https://hub.docker.com/r/jenkins/jenkins/)

Reply:
yes sorry, I didn't see it was still `jenkinsci`.